components/Filters/FilterConditionValues.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Select, ErrorZone } from 'react-components';
+import { noop } from 'proton-shared/lib/helpers/function';
+
+import EditConditionValue from './editor/EditConditionValue';
+import AddConditionValue from './editor/AddConditionValue';
+
+function FilterConditionValues({ condition, options, onDelete, onChangeCondition, onAdd, onEdit, error }) {
+    const hasError = (key) => (error.errors || []).includes(key);
+    return (
+        <>
+            <Select
+                options={options}
+                name="filterConditions"
+                className="mb1"
+                onChange={onChangeCondition}
+                defaultValue={condition.Comparator.value}
```

Comment:
What is the `defaultValue`? Even if you need to select a default value for the select you should manage that with his `value` when you initialize his state.